python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -79,6 +79,12 @@ class SkippedSensorRun(NamedTuple):
     existing_run: DagsterRun
 
 
+class BackfillRun(NamedTuple):
```

Comment:
Name here reads a little odd to me since "Run" does not mean the same thing here as in the other things that are returned, which definitely refer to DagsterRuns specifcially

some alternatives to consider:
BackfillSubmitResult / BackfillTickResult / BackfillSubmission